src/cf-notification/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-notification",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I think we should manually correct this back down to 1.0 before performing this release.

Reply:
I.e., 1.0 before this release, resulting in 1.0.1 with this release.

Reply:
I originally thought npm releases can't be unpublished (cf-notifications has been published at v1.1.0) but it looks like they can if they're [less than 24 hours old](http://blog.npmjs.org/post/141905368000/changes-to-npms-unpublish-policy). 👍 

So I'll unpublish 1.1.0 and change the above line to 1.0.0.